packages/react-focus/src/components/FocusZone/FocusZone.test.tsx

```diff
@@ -91,9 +91,6 @@ describe('FocusZone', () => {
     elementRefName: 'elementRef',
     testOptions: {
       'consistent-callback-names': { ignoreProps: ['onActiveElementChanged'] },
-      'consistent-callback-args': {
```

Comment:
why is this being removed ? AFAIR these callback restrictions should be left as is in v8 ?

Reply:
This component has only one callback prop `onActiveElementChanged ` and it is in `ignoreProps`. So consistent-callback-args was testing nothing here and I removed it.